pkgs/applications/networking/instant-messengers/slack/default.nix

```diff
@@ -94,12 +92,12 @@ in stdenv.mkDerivation {
     substituteInPlace $out/share/applications/slack.desktop \
       --replace /usr/bin/ $out/bin/ \
       --replace /usr/share/ $out/share/
-  '' + stdenv.lib.optionalString darkMode ''
+  '' + stdenv.lib.optionalString (theme != null) ''
     cat <<EOF >> $out/lib/slack/resources/app.asar.unpacked/src/static/ssb-interop.js
     document.addEventListener('DOMContentLoaded', function() {
     let tt__customCss = ".menu ul li a:not(.inline_menu_link) {color: #fff !important;}"
     $.ajax({
-        url: '${darkModeCssUrl}',
+        url: '${theme}/theme.css',
```

Comment:
@NeQuissimus hey, sorry, but how is this suppose to work now?
This would expand for a new version of `slack-dark` package to

```
https://myslackgroup.slack.com/nix/store/wcpbpwp161w5gvlchi2nnvra60l1dlzw-slack-theme-black/theme.css
```

This would never work AFAIK. It might work with `file://` prefix, but from what I remember `electron` will always block AJAX calls to local filesystem.

Reply:
Also since url now has postfix of `/theme.css` always I would be unable to pass my own custom URL as a workaround sadly. :(

Reply:
Created a fix for this https://github.com/NixOS/nixpkgs/pull/65156